pkg/builtinchecks/yamls/docker-sock.yaml

```diff
@@ -0,0 +1,7 @@
+name: "docker-sock"
+description: "Alert on deployments with docker.sock mounted on containers"
+remediation: "Ensure the Docker socket is not mounted inside any containers."
```

Comment:
For this and the ones below, make the description/remediation more detailed. Ideally, a user should be able to look at them and understand why this check exists, what they can do to fix it, and also have a link that they can click on to learn more.

Reply:
updated with impact explanation and specific fix.